### ISSUE TITLE
Add timeout to verify command

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress#679]
Currently, the `cypress verify` command can just hang because we do not put a timeout on a test run. It would be better to add a a timeout, and if the test really timed out, show the error output.

**Comment by jennifer-shehane** [comment]
@Bkucera I think this is still the case concerning `cypress verify`, can you confirm?

**Comment by Bkucera** [comment]
@jennifer-shehane I'm not sure what timeout this issue is referring to. A timeout on `--smoke-test`?

**Comment by jennifer-shehane** [comment]
@bahmutov Any insight?

**Comment by Malavos** [comment]
@jennifer-shehane  hello there!

I believe it is the case. I'm not OP, but my cypress is running into the same error.
See this:

```
E:\Workfolder\mysecretprojectfolder>cypress open
It looks like this is your first time using Cypress: 3.3.1

 ×  Verifying Cypress can run C:\Users\Programação\AppData\Local\Cypress\Cache\3.3.1\Cypress
   → Cypress Version: 3.3.1
Cypress verification timed out.

This command failed with the following output:

C:\Users\Programação\AppData\Local\Cypress\Cache\3.3.1\Cypress\Cypress.exe --smoke-test --ping=813

----------

Command failed: C:\Users\Programação\AppData\Local\Cypress\Cache\3.3.1\Cypress\Cypress.exe --smoke-test --ping=813

----------

Platform: win32 (10.0.14393)
Cypress Version: 3.3.1
```

Any tips?

**Comment by Malavos** [comment]
Hello everyone,

Well, that was quick.
I digged a lot more, and found that [this issue](https://github.com/cypress-io/cypress/issues/4258) had the solution I needed. 

It was then, a quick fix. I just changed this:

`"cypress": "^3.3.1",`

To this:

`"cypress": "3.3.1",`

I have no idea why that worked, but it worked!
Should we close this issue?

**Comment by Bkucera** [comment]
fixed in #4080
